addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -66,7 +66,8 @@ class MicrosoftSync(models.AbstractModel):
     _description = "Synchronize a record with Microsoft Calendar"
 
     microsoft_id = fields.Char('Microsoft Calendar Id', copy=False)
-    need_sync_m = fields.Boolean(default=True, copy=False)
+    need_sync_m = fields.Boolean(default=True, copy=False,
+        help="This field helps to known when an event need to be resync")
```

Comment:
when a "microsoft" event needs to be resynced. Even if we are in microsoft module, this is always better to be more explicit :)